lib/text/vtt_text_parser.js

```diff
@@ -219,6 +222,75 @@ shaka.text.VttTextParser = class {
     return cue;
   }
 
+  /**
+   * Parses a WebVTT styles from the given payload.
+   *
+   * @param {string} payload
+   * @param {!shaka.text.Cue} rootCue
+   */
+  static parseCueStyles(payload, rootCue) {
+    const xmlPayload = '<span>' + payload + '</span>';
+    const element = shaka.util.XmlUtils.parseXmlString(xmlPayload, 'span');
+    if (element) {
+      /** @type {Array.<!shaka.extern.Cue>} */
+      const cues = [];
+      const VttTextParser = shaka.text.VttTextParser;
+      const childNodes = [...element.childNodes];
```

Comment:
This confused me for a moment, until I realized that childNodes is Array-like, but not an Array. 
 If I'm not mistaken, this would convert it.  Was that your goal?

I don't see any need to do that, since you only access `[0]` and iterate, both of which should work fine on `IArrayLike`.